client/ayon_core/hosts/max/plugins/load/load_max_scene.py

```diff
@@ -14,6 +16,56 @@
 from ayon_core.pipeline import get_representation_path, load
 
 
+class MaterialDupOptionsWindow(QtWidgets.QDialog):
+
+    def __init__(self, material_options):
+        super(MaterialDupOptionsWindow, self).__init__()
+        self.setWindowFlags(self.windowFlags() | QtCore.Qt.FramelessWindowHint)
+
+        self.material_option = None
+        self.material_options = material_options
+
+        self.widgets = {
+            "label": QtWidgets.QLabel(
+                "Select material duplicate options before loading the max scene."),
+            "material_options_list": QtWidgets.QListWidget(),
+            "warning": QtWidgets.QLabel("No material options selected!"),
+            "buttons": QtWidgets.QWidget(),
+            "okButton": QtWidgets.QPushButton("Ok"),
+            "cancelButton": QtWidgets.QPushButton("Cancel")
+        }
+        for option in material_options.values():
+            self.widgets["material_options_list"].addItem(option)
+        # Build buttons.
+        layout = QtWidgets.QHBoxLayout(self.widgets["buttons"])
+        layout.addWidget(self.widgets["okButton"])
+        layout.addWidget(self.widgets["cancelButton"])
+        # Build layout.
+        layout = QtWidgets.QVBoxLayout(self)
+        layout.addWidget(self.widgets["label"])
+        layout.addWidget(self.widgets["material_options_list"])
+        layout.addWidget(self.widgets["buttons"])
+
+        self.widgets["okButton"].pressed.connect(self.on_ok_pressed)
+        self.widgets["cancelButton"].pressed.connect(self.on_cancel_pressed)
+        self.widgets["material_options_list"].itemPressed.connect(
+            self.on_material_optionsPressed)
+
+    def on_material_optionsPressed(self, item):
```

Comment:
I think code style wise this should be:

```suggestion
    def on_material_options_pressed(self, item):
```